src/ixdat/calculators/ms_calculators.py

```diff
@@ -0,0 +1,829 @@
+        F,
+        name=None,
+        cal_type=None,
+    ):
+        super().__init__()
+        self.name = name or f"{mol}@{mass}"
+        self.mol = mol
+        self.mass = mass
+        self.cal_type = cal_type
+        self.F = F
+
+    def __repr__(self):
+        return (
+            f"{self.__class__.__name__}(name={self.name}, mol={self.mol}, "
+            f"mass={self.mass}, F={self.F})"
+        )
+
+    @property
+    def color(self):
+        return STANDARD_COLORS[self.mass]
+
+    @classmethod
+    def from_siq(cls, siq_cal_point):
+        return cls(
+            name=siq_cal_point.mol + "@" + siq_cal_point.mass,
+            mol=siq_cal_point.mol,
+            mass=siq_cal_point.mass,
+            cal_type=siq_cal_point.F_type,
+            F=siq_cal_point.F,
+        )
+
+    def to_siq(self):
+        if not plugins.use_siq:
+            raise QuantificationError(
+                "`MSCalPoint.to_siq` only works when using "
+                "`spectro_inlets_quantification` "
+                "(`ixdat.options.activate_siq()`). "
+                "For native ixdat MS quantification, use `MSMeasurement.calibrate`"
+            )
+        return plugins.siq.CalPoint(
+            mol=self.mol,
+            mass=self.mass,
+            F=self.F,
+            F_type=self.cal_type,
+        )
+
+
+class MSCalibration(Calculator):
+    """Class for mass spec calibrations.
+
+    This is a simple version, just implementing sensitivity facotrs.
+    Consider using the more powerful spectro_inlets_quantification package instead.
+    """
+
+    calculator_type = "MS calibration"
+
+    extra_linkers = {"ms_calibration_results": ("ms_cal_results", "ms_cal_result_ids")}
+    child_attrs = [
+        "ms_cal_results",
+    ]
+
+    def __init__(
+        self,
+        mol=None,
+        mass=None,
+        F=None,
+        name=None,
+        date=None,
+        tstamp=None,  # FIXME: No need to have both a date and a tstamp?
+        setup=None,
+        ms_cal_results=None,
+        technique="MS",
+        measurement=None,
+    ):
+        """
+        Args:
+            name (str): Name of the ms_calibration
+            date (str): Date of the ms_calibration
+            setup (str): Name of the setup where the ms_calibration is made
+            ms_cal_results (list of MSCalResult): The mass spec calibrations
+            measurement (MSMeasurement): The measurement used to make the calibration
+        """
+        if plugins.use_siq:
+            warnings.warn(
+                "spectro_inlets_quantification is active but you are making a native "
+                "ixdat MSCalibration. It's probably best not to mix the two!"
+            )
+        super().__init__(
+            name=name,
+            technique=technique,
+            tstamp=tstamp,
+            measurement=measurement,
+        )
+        if mol and mass and (F is not None):
+            if ms_cal_results:
+                raise QuantificationError(
```

Comment:
Can you improve the docstring so it's clear that there are these two options on how to generate an `MSCalibration`?

Reply:
Done.